### DISCUSSION TITLE
Keep Window up for Auto Types?

### DISCUSSION THREAD

**Discussion by RT483b** [keepassxreboot/keepassxc#8942] [Q&A] [answered]
KeePassXC - Version 2.7.1,   Revision: 5916a8f,  Operating system: Windows 10 Version 2009, CPU architecture: x86_64
Can't find how to do this, maybe put as future request? Some site login logistics require individual screens for User and Password. So have to initiate separate Auto types for User and Password. But after I initiate the User auto type, KeePassXC minimizes to tray. Would be nice to keep it's window up until I want it minimized. I tried View->Always On Top, but it still minimizes after user auto type. Thanks for any help or future implementation.

**Comment by droidmonkey** [comment, marked as answer]
Use Global Auto-Type shortcut -> https://keepassxc.org/docs/KeePassXC_UserGuide.html#_performing_global_auto_type

We can't leave the window up, otherwise the typing wouldn't work...

**Reply by RT483b** [reply]
Thanks, I was using Entry-Level Auto-Type for each pertinent webpage of the login procedure. I didn't realize the KeePassXC app window needs to minimize after the User auto type. I'll look into the Global window title thing, otherwise I'll just have to click it back up from the sys tray to initiate the Password auto type.
